dash-pipeline/Makefile

```diff
@@ -412,6 +415,19 @@ run-saithrift-client-bash:
 # 	    --build-arg available_processors=$(shell nproc) \
 # 		dockerfiles
 
+# Builder, has base packages to make client docker
+docker-dash-p4c-bldr:
```

Comment:
You also need to make target docker-pull-dash-p4c-bldr which will be called in CI scripts. (We `docker pull` deliberately to make a failure more obvious.)
More importantly, you need to manually build and push this image to dockerhub for CI to pass, because the Dockerfile which uses this as a base cannot be found outside your PC environment, hence this CI failure:
https://github.com/Azure/DASH/actions/runs/3208306232/jobs/5244034990#step:3:570
I just made a new repo [chrissommers/dash-p4c-bldr](https://hub.docker.com/repository/docker/chrissommers/dash-p4c-bldr) and added you as a collaborator. You can use this until we migrate to ACR.